Scripts/script-Append.yml

```diff
@@ -0,0 +1,33 @@
+commonfields:
+  id: Append
+  version: -1
+name: Append
+script: |
+  result = demisto.args()["prefix"] + demisto.args()["suffix"]
+  res = {
+      'ContentsFormat': formats['text'],
+      'Type': entryTypes['note'],
+      'Contents': result,
+      'ReadableContentsFormat': formats['text'],
+      'HumanReadable': result,
```

Comment:
I would not add entry in the war room only for append strings, too noisy and not relevant for the investigation.
Should have output only